tree/tree/src/TLeafD32.cxx

```diff
@@ -205,8 +204,8 @@ void TLeafD32::Streamer(TBuffer &R__b)
    if (R__b.IsReading()) {
       R__b.ReadClassBuffer(TLeafD32::Class(), this);
 
-      if (fTitle.Contains("["))
-	 fElement = new TStreamerElement(Form("%s_Element", fName.Data()), fTitle.Data(), 0, 0, "Double32_t");
+      if (fElement && fElement->GetTitle().Contains("["))
```

Comment:
This should always be 'false' as `fElement` is marked transient ....

Reply:
hmm true...

Reply:
What do you think of changing:

fTitle = type;

with 

fTitle += "/" + type;

Reply:
Your message and mine criss-crossed.  I agree (but devils is in the details, including making sure to be able to read 'old' files with the previous notation (those old files can not be expected to be readable for the parts that have a variable size array of double32 but regular double32 should be readable).

Related: we need to also add a test for fixed size array.